### DISCUSSION TITLE
pg_net http_post not calling the function

### DISCUSSION THREAD

**Discussion by MicrorealHQ** [supabase/supabase#22481] [Questions]
Is there anyone got this issue?
I am trying to call my function with the command like this but when I checked supabase log, I couldn't see any log/invocation

```
select
      net.http_post(
          url:='https://supabase-id.supabase.co/functions/v1/funct-test/funct-test-inner',
          headers:='{"Content-Type": "application/json", "x-api-key": "key"}'::jsonb,
          body:=concat('{"limit": 20}')::jsonb
      ) as request_id;
```

Noted: I have tried to test in postman with the same request like this, the function is called normally

**Comment by GaryAustin1** [comment]
You may need the authorization header.
https://supabase.com/docs/guides/functions/quickstart#invoking-edge-functions-locally
Covers calling and mentions the option to turn off the jwt requirement.
Just a guess though.

**Reply by MicrorealHQ** [reply]
I deployed with ` --no-verify-jwt ` option, so jwt shouldn't be required (I tested from postman and function is called normally)

**Reply by GaryAustin1** [reply]
Any errors in the Postgres logs?

**Comment by j-mendez** [comment]
> Is there anyone got this issue? I am trying to call my function with the command like this but when I checked supabase log, I couldn't see any log/invocation
> 
> ```
> select
>       net.http_post(
>           url:='https://supabase-id.supabase.co/functions/v1/funct-test/funct-test-inner',
>           headers:='{"Content-Type": "application/json", "x-api-key": "key"}'::jsonb,
>           body:=concat('{"limit": 20}')::jsonb
>       ) as request_id;
> ```
> 
> Noted: I have tried to test in postman with the same request like this, the function is called normally

Seen this issue, also having trouble doing a simple POST from a trigger.

**Reply by storelord** [reply]
I can only do http_posts without headers. I haven't had any luck with pg_net either.

**Reply by GaryAustin1** [reply]
@storelord You should probably start a new discussion or better use the Discord and provide more details on what you are doing.  pg_net and the http extension both work and many, many users are using them including webhooks.
